metagraph/src/annotation/annotation_converters.cpp

```diff
@@ -1195,12 +1175,19 @@ void convert_to_row_diff<IntRowDiffDiskAnnotator>(
 
     std::mutex mtx;
 
+    size_t max_val = 0;
+
     ColumnCompressed<>::load_columns_and_values(
             files,
             [&](size_t column_index, const std::string &label, std::unique_ptr<bit_vector> &&column,
                 sdsl::int_vector<> &&values) {
+                    size_t local_max_val = *std::max_element(values.begin(), values.end());
```

Comment:
Can `values` be empty?

Reply:
Oh, nice catch, thanks! Fixed with 514b256e267bef93828e40ee62a6c281c2e74f9f